tests/ui/conftest.py

```diff
@@ -63,12 +63,30 @@ def user(base_url, fxa_account, jwt_token):
 
 
 @pytest.yield_fixture(scope='session')
-def firefox_path():
-    scraper = FactoryScraper('release', version='46.0.1')
+def firefox_path(tmpdir_factory):
+    tmp_dir = tmpdir_factory.mktemp('firefox')
+    scraper = FactoryScraper('release', version='latest')
```

Comment:
The `Scraper` base class has a `download` keyword argument, which you can pass the temporary directory to: https://github.com/mozilla/mozdownload/blob/64c10cc75be544b269daa3bca94e2d517fc72fd1/mozdownload/scraper.py#L76